bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/TableProvider.java

```diff
@@ -44,6 +46,8 @@
  */
 public class TableProvider {
 
+    private static final String UNASSIGNABLE_UNIONTYPE_EXCEPTION =
```

Comment:
Shall we move this to BallerinaErrorReasons and use BallerinaErrors.createError to throw runtime errors?

Reply:
This is a inner implementation and all other places we have return a Ballerina exception.